extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.css

```diff
@@ -212,6 +212,35 @@
   cursor: pointer !important;
 }
 
+.i-amphtml-lbg-thumbnail-timestamp-container {
+  background-color: #292D33;
+  color: #ffffff;
+  position: absolute;
+  bottom: 10px;
+  left: 10px;
+  height: 20px;
+  border-radius: 2px;
+  opacity: 0.8;
+  width: 20px;
+}
+
+.i-amphtml-lbg-thumbnail-timestamp-container.i-amphtml-lbg-has-timestamp {
+  font-size: 12px;
+  padding: 0 5px 0 18px;
+  line-height: 1.8;
```

Comment:
I think `line-height` requires a unit.

Reply:
Actually if you just specify a number, it translates to: 
> A number that will be multiplied with the current font-size to set the line height

Reply:
huh weird TIL